### PR TITLE
generate: support blkio related options

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao mashimiao.fnst@cn.fujitsu.com

### CODE REVIEW

File: generate/generate.go [hunk 1]
```diff
@@ -374,12 +374,84 @@ func (g *Generator) SetLinuxResourcesOOMScoreAdj(adj int) {
...
 }
 
+// SetLinuxResourcesBlockIOLeafWeight sets g.spec.Linux.Resources.BlockIO.LeafWeight.
+func (g *Generator) SetLinuxResourcesBlockIOLeafWeight(weight uint16) {
+	g.initSpecLinuxResourcesBlockIO()
+	g.spec.Linux.Resources.BlockIO.LeafWeight = &weight
+}
+
+// SetLinuxResourcesBlockIOLeafWeightDevice sets g.spec.Linux.Resources.BlockIO.WeightDevice.LeafWeight.
+func (g *Generator) SetLinuxResourcesBlockIOLeafWeightDevice(major int64, minor int64, weight uint16) {
```
Comment (Mashimiao) [inline generate/generate.go]: Yes, I will update this PR in a few days.

**Comment by Mashimiao** [comment]
ping @wking  @mrunalp  @liangchenye    fixed and rebased.

**Comment by Mashimiao** [comment]
Rebased. @mrunalp  @wking  @liangchenye  @hqhq  PTAL

**Comment by liangchenye** [comment]
LGTM

**Comment by q384566678** [comment]
LGTM

**Pull request merged**